extensions/amp-user-notification/validator-amp-user-notification.protoascii

```diff
@@ -52,4 +52,7 @@ tags: {  # <amp-user-notification>
       allow_empty: false
     }
   }
+  attrs: {
+    name: "enctype"
```

Comment:
We need to specify the explicit values allowed. In this case, just the `value: 'application/x-www-form-urlencoded'`. /cc @honeybadgerdontcare to verify.

Reply:
@jridgewell That's correct.

```
attrs: {
  name: "enctype"
  value: "application/x-www-form-urlencoded"
}
```